lib/main.dart

```diff
@@ -60,6 +62,15 @@ class _MyHomePageState extends State<MyHomePage> with TickerProviderStateMixin {
     @required this.store,
   });
 
+  Future<Null> getLocation() async {
+    LocationService location = new LocationService();
```

Comment:
var location = ...

Reply:
@pacane Do you mean: final LocationService... ?

Reply:
or just `final`?

Reply:
Just final